src/views/CountryThreeWProjects/Map/index.tsx

```diff
@@ -0,0 +1,399 @@
+                >
+                    <MapLayer
+                        layerKey="points-halo-circle"
+                        onClick={handlePointClick}
+                        layerOptions={{
+                            type: 'circle',
+                            paint: redPointHaloCirclePaint,
+                        }}
+                    />
+                    <MapLayer
+                        layerKey="points-circle"
+                        layerOptions={{
+                            type: 'circle',
+                            paint: redPointCirclePaint,
+                        }}
+                    />
+                </MapSource>
+            )}
+            {emergencyGeo && (
+                <MapSource
+                    sourceKey="emergency-points"
+                    sourceOptions={sourceOption}
+                    geoJson={emergencyGeo}
+                >
+                    <MapLayer
+                        layerKey="points-halo-circle"
+                        onClick={handlePointClick}
+                        layerOptions={{
+                            type: 'circle',
+                            paint: bluePointHaloCirclePaint,
+                        }}
+                    />
+                    <MapLayer
+                        layerKey="points-circle"
+                        layerOptions={{
+                            type: 'circle',
+                            paint: bluePointCirclePaint,
+                        }}
+                    />
+                </MapSource>
+            )}
+            <MapBounds
+                // FIXME: use defined constants
+                duration={1000}
+                padding={10}
+                bounds={countryBounds}
+            />
+            {multiTypeGeo && (
+                <MapSource
+                    sourceKey="multi-points"
+                    sourceOptions={sourceOption}
+                    geoJson={multiTypeGeo}
+                >
+                    <MapLayer
+                        layerKey="points-halo-circle"
+                        onClick={handlePointClick}
+                        layerOptions={{
+                            type: 'circle',
+                            paint: orangePointHaloCirclePaint,
+                        }}
+                    />
+                    <MapLayer
+                        layerKey="points-circle"
+                        layerOptions={{
+                            type: 'circle',
+                            paint: orangePointCirclePaint,
+                        }}
+                    />
+                </MapSource>
+            )}
+            {clickedPointProperties?.lngLat && selectedDistrictProjectDetail && (
+                <MapPopup
+                    coordinates={clickedPointProperties.lngLat}
+                    onCloseButtonClick={handlePointClose}
+                    heading={(
+                        selectedDistrictProjectDetail[0].project_district_detail.name
```

Comment:
Are we sure we will always have at least one element?

Reply:
Yes

Reply:
Let's add a NOTE